PowerEditor/src/NppIO.cpp

```diff
@@ -613,7 +612,27 @@ bool Notepad_plus::doReload(BufferID id, bool alert)
 
 	// Once reload is complete, activate buffer which will take care of
 	// many settings such as update status bar, clickable link etc.
-	activateBuffer(id, currentView(), true);
+	if ( ((currentView() == MAIN_VIEW) && mainVisible) || ((currentView() == SUB_VIEW) && subVisible))
+	{
+		activateBuffer(id, currentView(), true);
+	}
+	else

```

Comment:
`if ( ((currentView() == MAIN_VIEW) && mainVisible) || ((currentView() == SUB_VIEW) && subVisible))` means:
the main Sintilla is focused and the buffer in argument of function is activated in main scintilla
or
the sub Sintilla is focused and the buffer in argument of function is activated in sub-scintilla

so what's the situation of `else` ?

Reply:
@donho 
It's a less common case, where the reloaded tab/buffer is not in the active selected view.

Due to the `Notepad_plus::doReload()` design, where the original `activateBuffer(id, currentView(), true)` takes the final care of the reloaded tab/buffer setup, if the `currentView()` and the reloaded tab/buffer view does not match, the following `DocTabView::activateBuffer(BufferID buffer)` fails (returns 0xffffffff due to `DocTabView::getIndexByBuffer(BufferID id)` fail) and the `Notepad_plus::activateBuffer(BufferID id, int whichOne, bool forceApplyHilite)` code will not do the requested setup job.